build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.0'
```

Comment:
I've noticed in the past that having this as 3.3.0 gives me problems whenever I try to use "R" (says like unresolved identifier). Did you have this problem? For example: R.layout.activity_main (in MainActivity.java).